utils.js

```diff
@@ -53,6 +59,24 @@ function post(url, jar, form) {
   return request(op).then(function(res) {return res[0];});
 }
 
+function postFormData(url, jar, form, qs) {
```

Comment:
If the only difference is including the `qs` in the form, I think it could be fine just added to `post`, yeah.